internal/databases/postgres/backup_time.go

```diff
@@ -0,0 +1,12 @@
+package postgres
+
+import "time"
+
+// BackupTime is used to sort backups by
+// latest modified time.
+type BackupTime struct {
```

Comment:
This logic is not Postgres-related. You can now use GenericMetadata to fetch `CreationTime` (StartTime) for any database. So I suggest to leave it in internal and use the GenericMetaFetcher to fetch the backup start time like this:

```
func doSmth(..., metaFetcher GenericMetaFetcher) {
...
meta, err := metaFetcher.Fetch(backupName, folder.GetSubFolder(utility.BaseBackupPath))
...
}
```